src/control/server/config_test.go

```diff
@@ -229,10 +250,10 @@ func TestServer_ConstructedConfig(t *testing.T) {
 				WithScmMountPoint("/mnt/daos/2").
 				WithScmClass("dcpm").
 				WithScmDeviceList("/dev/pmem0").
-				WithBdevClass("kdev").
-				WithBdevDeviceList("/dev/sdc", "/dev/sdd").
+				WithBdevClass("malloc").
```

Comment:
why change this? was this test broken before?

Reply:
Because the example config contains multiple bdev configs in the second server block. Before this PR, they were all uncommented and the config would load OK, with the last config being the one that "won". With strict parsing enabled, we have to omit those duplicate configs and so now it's the first example bdev config that is represented in the loaded config.